litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts

```diff
@@ -2,23 +2,11 @@ import { makeStyles } from '@material-ui/core/styles';
 
 // Component styles
 const useStyles = makeStyles((theme) => ({
-  rootContainer: {
-    display: 'flex',
-    flexDirection: 'column',
-    justifyContent: 'space-between',
-    overflowX: 'hidden',
-    msOverflowX: 'hidden',
-  },
-
   root: {
-    overflowY: 'hidden',
-    msOverflowY: 'hidden',
-    msOverflowStyle: 'none' /* Internet Explorer 10+ */,
     scrollbarWidth: 'none' /* Firefox */,
     '&::-webkit-scrollbar': {
       display: 'none' /* Safari and Chrome */,
     },
-    marginBottom: theme.spacing(2.5),
```

Comment:
Please add this, it was given for the styling as per figma, not for the tooltip.

Reply:
the main tag has consistent padding for every page. We can't add special padding for one single page.
If you feel the bottom padding is incorrect please change it for all pages. I think currently what we have is enough.

Reply:
It’s not a padding, it’s a margin @arkajyotiMukherjee and we don’t have any such thing as consistent margin for all pages, it is varying across pages. Please take a look at both figma as for the reason of adding this margin and also some of the files for example like

https://github.com/whitetiger1399/litmus/blob/0f42692e06c28b4a6673b27cbe3eb75ed04aa6c3/litmus-portal/frontend/src/pages/ChaosHub/styles.ts#L12

https://github.com/whitetiger1399/litmus/blob/0f42692e06c28b4a6673b27cbe3eb75ed04aa6c3/litmus-portal/frontend/src/pages/Community/styles.ts#L6

The community page was recently refactored by @whitetiger1399

Reply:
The current bottom margin we have is good enough I think you may verify the same from figma. Removing this will deviate the implementation from the design.

Reply:
I still stand by what I said...It needs to be consistent.
Add it [here](https://github.com/litmuschaos/litmus/blob/master/litmus-portal/frontend/src/containers/layouts/Scaffold/styles.ts) if you feel like it and not in individual pages.

Reply:
@whitetiger1399 can change it in the other pages as well.

Reply:
Yes either we change it and remove from all pages and control this centrally or we should keep the bottom margin here as well. Just removing the margin may be a good change @arkajyotiMukherjee and controlling it centrally as well but since the implementation is more apt. as per the design it would be good if we don’t remove it before the release. The graphs get to the bottom of the area and it looks like a UI bug. 

@whitetiger1399 either you can add this margin property back or please add necessary styling to the enclosing component. This is a negative change as of now, design wise.